psi4/driver/driver.py

```diff
@@ -82,6 +82,9 @@ def _find_derivative_type(ptype, method_name, user_dertype):
             raise ValidationError("_find_derivative_type: user_dertype should only be None or int!")
         dertype = user_dertype
 
+    if (core.get_global_option('INTEGRAL_PACKAGE') == 'ERD') and (dertype != 0):
+        raise ValidationError("INTEGRAL_PACKAGE ERD does not play nicely with gradients, so stopping.")
+
```

Comment:
So one could put in a check for LRC-DFT. Would look something like the below in driver.py. But would have many unnecessary halts because not all LRC are faulty– first one in dft1 is `UKS -1 2   wB97` whereas the cation had been just fine. And also there would be insufficient halts because if `energy('scf')` and setting the functional, then can't match up a name to test in the superfunc_list. So checking is just not clean.

```
 467     else:
 470             for ssuper in superfunc_list:
 471                 if ssuper.name().lower() == lowername:
 472                     if ssuper.is_c_lrc() or ssuper.is_x_lrc():
 473                         # stop in tracks
 474         return core.get_variable('CURRENT ENERGY')
```